utils/utils.rb

```diff
@@ -0,0 +1,67 @@
+module Fastlane
+  module Actions
+    class UtilsAction < Action
+      def self.run(params)
+        action = params[:action]
+
+        case action
+        when "is_branch_up_to_date"
```

Comment:
`utils.rb` is to general - please name it accordingly and remove all logic which is connected to `action` - no need in it - there is only one `is_branch_up_to_date ` action. It is unnecessary complication here - make it simpler. Less code - better.